R2API/ItemAPI.cs

```diff
@@ -583,7 +583,10 @@ public class ItemDisplayRuleDict {
         internal bool HasInvalidDisplays(out StringBuilder logger) {
             bool invalidDisplays = false;
             logger = new StringBuilder();
-            foreach(var (bodyName, rules) in Dictionary) {
+            foreach (var (bodyName, rules) in Dictionary) {
+                if (rules == null)
```

Comment:
I agree with needing to null check here, but what did people honestly expect when they add a body with `null` as the rules..

Reply:
https://github.com/risk-of-thunder/R2API/blob/1fbac1d0ceb32176bf99887a7645ae2750a6395f/R2API/ItemAPI.cs#L98 + it doesnt seem to make sense ?

Reply:
or actually, that null check seems to be happening too late anyway, as I don't think .Any is null checking ?

Reply:
This amount of nullchecking was a result of this conversation on the discord regarding a NRE Keroro was getting from ItemAPI.

https://discordapp.com/channels/562704639141740588/562704639569428506/985227168160317481
I supose some of the nullchecks are unecesary, but oh well

Reply:
its not what I said, the null check are fine, its just the rules == null which doesnt make sense, because the caller of that method just before is calling rules.Any on all of them

Reply:
Removed unecesary null check